packages/dynamic_color/example/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ description: Demonstrates how to use the dynamic_color package.
 publish_to: "none" # Remove this line if you wish to publish to pub.dev
 
 environment:
-  sdk: ">=2.17.0-0 <4.0.0"
+  sdk: ">=3.0.0 <4.0.0"
```

Comment:
I had to do this in order to have access to access to record destructing syntax.